PocoDoc/src/PocoDoc.cpp

```diff
@@ -351,7 +346,23 @@ class PocoDocApp: public Application
 		File file(path);
 		file.createDirectories();
 
-		DocWriter writer(_gst, path.toString(), config().getBool("PocoDoc.prettifyCode", false), _writeEclipseTOC);
+		bool searchIndex = false;
+
+		if (config().getBool("PocoDoc.searchIndex", false))
```

Comment:
this is fine, but there should also be possible to [specify it on the command line](https://github.com/vrabac6/poco/blob/21ed183dc1697784bd9d99cab4c18ee8a3c73be9/PocoDoc/src/PocoDoc.cpp#L142):